packages/terra-doc-template/src/ExampleTemplate.module.scss

```diff
@@ -3,7 +3,6 @@
     border: 1px solid 	#e8e8e8;
     border-radius: 3px;
     margin-bottom: 0.9375rem;
-    overflow: hidden;
```

Comment:
This is a little concerning, but I'm not seeing any issues browsing through the other doc examples. It might be worth double checking the dev site in IE 10 just to make absolutely certain this overflow isn't doing anything.

Also, the crazy space on the `border` property above cannot be unseen.